Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -496,19 +462,22 @@ class TabTrayController: UIViewController {
 
     @objc
     private func didTapMenu() {
-        let mvc = MenuViewController(withAppState: .TabTray(tabTrayState: self.tabTrayState), presentationStyle: .Popover)
+        let presentationStyle: MenuViewPresentationStyle = (self.traitCollection.horizontalSizeClass == .Compact && traitCollection.verticalSizeClass == .Regular) ? .Modal : .Popover
```

Comment:
Looks like it's not a popover in the mocks you mentioned above so modal all the way!